library/src/scripts/components/forms/select/overwrites/ClearIndicator.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * @author Stéphane LaFlèche <stephane.l@vanillaforums.com>
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import * as React from "react";
+
+import classNames from "classnames";
+import { ButtonBaseClass } from "@library/components/forms/Button";
+import { t } from "@library/application";
+import { clear } from "@library/components/Icons";
+
+export function ClearIndicator({ children, ...props }) {
```

Comment:
This should also be a class component. The `handleKeyDown` function is being recreated on every render in this setup.